src/views/world/view_missile.cpp

```diff
@@ -0,0 +1,44 @@
+#include "fiber_pool.hpp"
+#include "natives.hpp"
+#include "pointers.hpp"
+#include "services/players/player_service.hpp"
+#include "views/view.hpp"
+
+
+namespace big
+{
+	void view::missile()
+	{
+		ImGui::Text("Missle Launch");
+
+		static player_ptr victim = g_player_service->get_selected();
+		ImGui::SeparatorText("Victim");
+
+		if (ImGui::BeginCombo("##victim", victim->get_name()))
+		{
+			auto self = g_player_service->get_self();
+			if (ImGui::Selectable(self->get_name(), self->id() == victim->id()))
+				victim = self;
+			for (auto p : g_player_service->players() | std::ranges::views::values)
+			{
+				if (ImGui::Selectable(p->get_name(), p->id() == victim->id()))
+				{
+					victim = p;
```

Comment:
What does p represent here? Useless variable name.

Reply:
It was taken from squad_spawner 🤣 ill fix that